CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+# 0.4.0
+
+This version roughly corresponds with all the changes made to the Temporal polyfill as of the February 2022 TC39 Plenary meeting.
```

Comment:
Small nitpick: There wasn't a February meeting, just a January one :smile:

Reply:
I honestly haven't been paying attention, so I expected this to get corrected. Thanks!